Source/CDMarkdownParser.swift

```diff
@@ -168,8 +169,8 @@ open class CDMarkdownParser {
                                       value: paragraphStyle,
                                       range: range)
         var elements: [CDMarkdownElement] = escapingElements
-        elements.append(contentsOf: defaultElements)
         elements.append(contentsOf: customElements)
+        elements.append(contentsOf: defaultElements)
```

Comment:
I think if a developer wants to override some classes or introduce custom versions or the existing ones, they should be able to do that. This will allow the custom elements to be parsed first.

Reply:
Good point. I'll keep this change.

Reply:
I decided for the time being to keep this out. I would imagine a developer making a custom element should have a regex that differs from the default elements, thus kind of rendering order not as important. Please let me know if you have an example where that doesn't work and I'll look at it again.